plugin/trino-google-sheets/src/test/java/io/trino/plugin/google/sheets/TestGoogleSheets.java

```diff
@@ -78,4 +83,39 @@ public void testTableWithRepeatedAndMissingColumnNames()
         assertQuery("desc table_with_duplicate_and_missing_column_names", "SELECT * FROM (VALUES('a','varchar','','')," +
                 " ('column_1','varchar','',''), ('column_2','varchar','',''), ('c','varchar','',''))");
     }
+
+    // test polymorphic table function
+
+    @Test
+    public void testSheetQuerySimple()
```

Comment:
Could you add another test method having outer WHERE clause?

Reply:
Added another test with a where clause to validate column naming.